Faraday_Proxy_Tools/FaradayIO/deviceconfig.py

```diff
@@ -178,21 +247,30 @@ def update_gps(self, gps_boot_bitmask, latitude_str, latitude_dir_str, longitude
             self.gps_altitude = altitude_str
             self.gps_altitude_units = altitude_units_str
             self.gps_boot_bitmask = gps_boot_bitmask
+
+            return True
+
         else:
```

Comment:
You should be using the logging module already being used for this in flask. I would also be more granular, clearly indicate exactly what is wrong. Not a list of what could be wrong.